Misc/NEWS.d/next/Library/2024-06-09-19-53-11.gh-issue-120289.s4HXR0.rst

```diff
@@ -0,0 +1 @@
+Generate an unraisable exception if the external timer :mod:`cProfile` uses changes the profile context.
```

Comment:
I don't think this is correct any more.

Reply:
I changed that. I did not put implementation detail in it because I remember that we are not supposed to do that. So I just said I fixed the issue and mentioned the context.